3_javascript/javascript1/week1/javascript.js

```diff
@@ -0,0 +1,72 @@
+
+//Age-ify (A future age calculator)
+
+// let yearOfBirth = 1990
+// let yearFuture = 2050
```

Comment:
This code commented, but I would think this is a solution, right?

Reply:
yes, I forgot to un-comment them. now is done!